drupal/manifests/drupal-secrets.yaml

```diff
@@ -0,0 +1,11 @@
+#helm:generate helmc tpl -d tpl/values.toml -o manifests/drupal-secrets.yaml $HELM_GENERATE_FILE
+apiVersion: v1
+kind: Secret
+metadata:
+  name: drupal
```

Comment:
@prydonius @migmartri I've named the secret volume the same as the chart name. Do you think it should be named differently. Not that there is anything wrong with the current name.